expo-maps/src/Maps.tsx

```diff
@@ -17,13 +17,73 @@ const defaultNativeExpoMapViewProps: DefaultNativeExpoMapViewProps = {
 };
 
 export class ExpoMap extends React.Component<ExpoMapViewProps> {
+  private mapChildren() {
+    const childrenArray = React.Children.map(this.props.children, (child) => {
```

Comment:
I'm wondering what is correct policy overall about children objects. Should we handle (ignore) all possible incorrect input or meaningless children objects like `null`, `undefined`, `number` etc? Or maybe we should assume that programmer using our library read the docs and is only using valid children like `ExpoMarkers` and if so we can type them like this:
```suggestion
    const childrenArray = React.Children.map(this.props.children as ExpoMarker[] | ExpoMarker, (child) => {
```
and simplify validation process?
What is your opinion @lukmccall?

Reply:
In this case, perhaps the best approach would be to:
1. Ignore `null`, `undefined` (generally falsy values) because someone may want to conditionally put the marker as a child (`{showMarker ? <Marker ... /> : null}`) and that should be totally fine.
2. If the child is of unexpected/unsupported type, then ignore it as well but also print a warning — people should be aware that adding things like `View` or `Text` is not possible that way.